### PR TITLE
To upgrade fastjson to 1.2.72

### PR DESCRIPTION
**Pull request by AVANZHAOYONG**
### Ⅰ. Describe what this PR did
To resolve security issues, update fastJSON version to 1.2.72

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
Deserialize remote code execution vulnerabilities [fastjson security_update](https://github.com/alibaba/fastjson/wiki/security_update_20200601)

### Ⅳ. Describe how to verify it
update fastJSON version to 1.2.72.
And i've already verified that, replace FastJSON 1.2.72 without any other impact

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -66,7 +66,7 @@
         <dubbo.version>2.7.0</dubbo.version>
         <dubbo.alibaba.version>2.6.5</dubbo.alibaba.version>
         <sofa.rpc.version>5.5.3</sofa.rpc.version>
-        <fastjson.version>1.2.60</fastjson.version>
+        <fastjson.version>1.2.70</fastjson.version>
```
Comment (a364176773) [inline bom/pom.xml:69]: why not use the latest edition?

[the remark this answers is not in the document]
Comment (AVANZHAOYONG) [reply bom/pom.xml]: Has been updated and verified.

[the remark this answers is not in the document]
Comment (AVANZHAOYONG) [reply bom/pom.xml]: > why not use the latest edition?

Has been updated and verified.

**Comment by a364176773** [comment]
pr考虑到目前合并，可能1.4的时候fastjson又发版了，所以这个pr可能会放在seata1.4发版前进行review。
所以请耐心等待。

**Pull request closed without merging**